index.d.ts

```diff
@@ -10,6 +10,8 @@ declare module 'docker-progress' {
 
 		public aggregateProgress(count: number, onProgress: ProgressCallback): ProgressCallback[];
 
+		public modem: any;
```

Comment:
How does that work? The fact that it's declared doesn't make it available and I don't think `@modem` is ever assigned in the coffee file.